src/models/order.js

```diff
@@ -0,0 +1,57 @@
+import Backbone from 'backbone';
+import QuoteList from 'collections/quote_list';
+
+const Order = Backbone.Model.extend({
+  defaults: {
+    symbol: '',
+    targetPrice: '',
+    buy: '',
+  },
+  initialize(attributes) {
+    this.buy = attributes.buy;
+    this.targetPrice = attributes.targetPrice
+    this.quote = attributes.activeQuote;
```

Comment:
I like that the `Order` knows which quote it's for. It's possible to achieve the same behavior through the bus, but the resulting code ends up being much more complex and hard to read.